config.js

```diff
@@ -674,13 +674,38 @@ var config = {
         // userRegion: "asia"
     },
 
+    // Array<string> of disabled sounds.
+    // Possible values:
+    // - PARTICIPANT_JOINED_SOUND,
+    // - PARTICIPANT_LEFT_SOUND,
+    // - RAISE_HAND_SOUND,
+    // - INCOMING_MSG_SOUND,
+    // - E2EE_OFF_SOUND
+    // - E2EE_ON_SOUND
+    // - OUTGOING_CALL_START_SOUND
+    // - OUTGOING_CALL_RINGING_SOUND
+    // - OUTGOING_CALL_REJECTED_SOUND
+    // - OUTGOING_CALL_EXPIRED_SOUND
+    // - KNOCKING_PARTICIPANT_SOUND
+    // - NO_AUDIO_SIGNAL_SOUND
+    // - NOISY_AUDIO_INPUT_SOUND
+    // - RECORDING_ON_SOUND
+    // - RECORDING_OFF_SOUND
+    // - LIVE_STREAMING_ON_SOUND
+    // - LIVE_STREAMING_OFF_SOUND
+    // - TALK_WHILE_MUTED_SOUND
+    // disabledSoundIds: [],
```

Comment:
Let's call this `disabledSounds` the fact that we pass ids is just an implementation detail.